src/SnakesAndLadders.java

```diff
@@ -0,0 +1,103 @@
+import java.util.Scanner;
+
+public class SnakesAndLadders {
+
+    int Ladders[],Snakes[],LaddersEnd[],SnakesEnd[];
+
+    //Boolean class variable to check if the game is over or not
+    boolean gameOver = false;
+
+    //Scanner class object to take input from the user
+    Scanner sc = new Scanner(System.in);
+
+    //Array to store the positions of the players
+    int playerPositions[];
+
+    //Method to start the snakes and ladders game
+    public void snakesAndLadders() {
+
+        //Initializing the game over variable to false
+
+        gameOver = false;
+
+        //Initializing the Ladders and Snakes array
+        Ladders = new int[]{1,4,8,21,28,50,71,80};
```

Comment:
Can this be converted into class as they have properties related to it i.e. head or tail and initialise that?